examples/mautic-example-nginx-ssl-managedDB/docker-compose.yaml

```
@@ -0,0 +1,34 @@
+version: "2"
+services:
+  mtcpilotdev9:
+    container_name: mtcpilotdev9
+    restart: unless-stopped
+    image: mautic/mautic:v3-apache
+    ports:
+      - "127.0.0.1:80:80"
+    environment:
+      - MAUTIC_DB_HOST=195.154.70.120:54788
+      - MYSQL_PORT_3306_TCP=54788
+      - MAUTIC_DB_USER=bstadmin
```

We should assume that folk may not realise that they should edit these fields - let's make it more obvious

```suggestion
      - MAUTIC_DB_USER=INSERT_YOUR_DB_USERNAME_HERE
```